README.md

```diff
@@ -40,33 +40,33 @@ app.middleware({logger: LoggerToken}, ({logger}) => {
     if (__NODE__) logger.info(`Received request at ${ctx.url}`);
     else logger.info(`Pageload at ${ctx.url}`);
     return next();
-  }
+  };
 });
 ```
 
-### Setup 
+### Setup
 
 ```js
 import App from 'fusion-core';
 import winston from 'winston';
 import UniversalEvents from 'fusion-plugin-universal-events';
-import UniversalLogger, {UniversalLoggerConfigToken} from 'fusion-plugin-universal-logger';
+import UniversalLogger, {
+  UniversalLoggerConfigToken,
+} from 'fusion-plugin-universal-logger';
 
 export default () => {
   const app = new App(<div>Hello</div>);
-  app.register(UniversalEventsToken, UniversalEvents)
+  app.register(UniversalEventsToken, UniversalEvents);
   app.register(LoggerToken, UniversalLogger);
-  if (__NODE__) { 
+  if (__NODE__) {
     // optional winston configuration
     const config = {
-      transports: [
-        new winston.transports.File({filename: 'logs.log'}),
-      ],
+      transports: [new winston.transports.File({filename: 'logs.log'})],
     };
     app.register(UniversalLoggerConfigToken, config);
   }
   return app;
-}
+};
```

Comment:
https://mislav.net/2010/05/semicolons/

Reply:
I agree, remove all semicolons! (This was prettier though...)